docs/ERROR_TRACKING.md

```
@@ -0,0 +1,238 @@
+SENTRY_PROJECT=your_project_name
+
+# Optional: Control error reporting
+NEXT_PUBLIC_SENTRY_ENVIRONMENT=production
+SENTRY_AUTH_TOKEN=your_auth_token_here
+```
+
+### 5. Update .gitignore
+
+Ensure sensitive Sentry files are not committed:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)